vtex-sites/storecomponents.store
# Refactor | Search Context

## What's the purpose of this pull request?
Refactors search's internal logic. For more info, look at this [PR](https://github.com/vtex/faststore/pull/684)

## How to test it?
Open any search page (plp). You must not see any difference from what's on production right now as this PR only changes internal logic and don't really make any different behavior
